core/templates/dev/head/components/create_exploration_button_directive.html

```diff
@@ -20,3 +20,141 @@
     {% endif %}
   </div>
 </script>
+<script type="text/ng-template" id="modals/createExplorationOrCollection">
```

Comment:
modals/createActivity ('activity' is the name we give to something that could be either a collection or an exploration)